apps/remote_control/lib/lexical/remote_control/plugin.ex

```diff
@@ -0,0 +1,95 @@
+defmodule Lexical.RemoteControl.Plugin do
+  alias Lexical.Document
+  alias Lexical.RemoteControl
+  alias Lexical.RemoteControl.Api.Messages
+  alias Lexical.RemoteControl.Plugin
+
+  require Logger
+  import Messages
+
+  def on_module_updated(plugin_module) do
+    if Keyword.has_key?(plugin_module.__info__(:attributes), :lexical_plugin) do
+      current_plugiuns = get()
+
+      unless exists?(current_plugiuns, plugin_module) do
+        plugin_module.init()
```

Comment:
We must ensure that `init/0` is only called after the project has been compiled. Can the design here achieve this goal?